src/app/apps/rup/mapa-camas/sidebar/cama-detalle/cama-detalle.component.ts

```diff
@@ -83,6 +90,46 @@ export class CamaDetalleComponent implements OnInit {
             pluck('acciones'),
             map(acciones => acciones.filter(acc => acc.tipo === 'nuevo-registro'))
         );
+
+        const HOY = moment().toDate();
+        this.historial$ = this.mapaCamasService.fecha2.pipe(
```

Comment:
Todo esto se reemplaza con este observable: `this.mapaCamaService.historialInternacion$ `. La internación debería tener un solo movimiento, no importa la fecha. Y de yapa debería ser el que dice extras.ingreso = true.